lib/playdate/games/sudoku/seeds/seeds.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Playdate.Games.Sudoku.Seeds do
+  @moduledoc """
+  Reads files with seeding data to generate sudoku games to be solved and their corresponding solutions
+  """
+
+  @doc """
+  Reads seeding files and generates a sudoku game and its solution
+  Index from `games_to_solve.txt` has to match the index from the `games-solved.txt`
+  """
+  @spec get_game() :: {tuple(), tuple()}
+  def get_game do
+    games = treat_data("lib/playdate/games/sudoku/seeds/games_to_solve.txt")
+    solutions = treat_data("lib/playdate/games/sudoku/seeds/games_solved.txt")
```

Comment:
```suggestion
    games = :playdate |> :code.priv_dir() |> Path.join("sudoku/seeds/games_to_solve.txt") |> treat_data()
    solutions :playdate |> :code.priv_dir() |> Path.join("sudoku/seeds/games_solved.txt") |> treat_data()
```

We need to store these files in the `priv` directory.  The reason being that we can't rely on the actual source code of the app (and thus the `lib` folder) being present in the deployed application.  All the Elixir and Erlang files for the app and it's dependencies will wind up just being in a single folder of `.beam` files containing just the BEAM bytecode for each module.  The `priv` folder is used to work around this by providing a location for an OTP application to store it's non-source code assets which it can reference when deployed.